Makefile

```diff
@@ -115,16 +115,26 @@ tidy: #HELP Update dependencies.
 	# Force tidy to use the version already in go.mod
 	$(Q)go mod tidy -go=$(GOLANG_VERSION)
 
+.PHONY: catalogd-manifests #EXHELP Generate catalogd code and catalogd manifests.
+KUSTOMIZE_CRDS_DIR := catalogd/config/base/crd/bases
+KUSTOMIZE_RBAC_DIR := catalogd/config/base/rbac
+KUSTOMIZE_WEBHOOKS_DIR := catalogd/config/base/manager/webhook
+catalogd-manifests: $(CONTROLLER_GEN) 
+	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
```

Comment:
This codegen using `hack/boilderplate.go.txt` should move to the `generate` target. I'm guessing we actually only need to call this once for the entire repo, so we may be able to just delete this. 

You can verify by deleting the catalogd generated deepcopy code and re-running `make generate` from the root Makefile.

Reply:
Fixed in https://github.com/operator-framework/operator-controller/commit/d06df7ff15ac685729983f5fc8890a923da1aeea